### ISSUE TITLE
Can you implement this suggestion!!!

### ISSUE THREAD

**Issue by RGitHuben** [elsa-workflows/elsa-core#4049]
I work with several BPMS systems and your work is awesome but the designer doesn't implement BMPS(by general meaning).
Could you use https://bpmn.io/toolkit/bpmn-js? It's free and can change your project to a big one.
It's derived from CAMUNDA and we can use the Custom Task for our custom code.

**Comment by sfmskywalker** [comment]
Yep, BPMN 2.0 is on the roadmap and likely one of the next first things to pickup post Elsa 3.0 release.

**Comment by sfmskywalker** [comment]
PS. As a general suggestion, please use titles for issues that summarize the contents. For example, a good title for this issue would be: “BPMN Support”.

Another kind suggestion is to do a preliminary search to see if there’s an existing issue already (which you’ll see there is).

Regardless, thank you for the suggestion!
